components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/impl/FederatedAuthenticatorConfigBuilderFactory.java

```diff
@@ -435,6 +466,23 @@ private static void resolveEndpointConfiguration(FederatedAuthenticator authenti
         }
     }
 
+    private static String[] resolveAuthenticatorTags(FederatedAuthenticatorConfig config) {
```

Comment:
Is there a requirement to return the tags as an array ? Can't we return a List ?